mti_nma/steps/nma/nma_viz.py

```diff
@@ -63,7 +75,16 @@ def color_vertices_by_magnitude(
         Filepath to output file of colored mesh object (.blend)
     """
 
-    args = f"-i {path_input_mesh} -o {path_output} -m  {mode} -v {path_vmags}"
-    cmd = f"{path_blender} -b -P color_vertices -- {args}"
+    args = f"-i {path_input_mesh} -o {path_output} -m {mode} -v {path_vmags}"
+    python_callable = str(
+        Path(__file__).parent.parent.parent / "bin" / "color_vertices.py"
+    )
+    cmd = f"{path_blender} -b -P {python_callable} -- {args}"
     p = subprocess.Popen(cmd, shell=True, executable="/bin/bash")
-    p.terminate()
+
+    # If an error occurs, log it as output to the terminal
+    if p.stderr is not None:
+        log.info(p.stderr)
+
+    # Closes subprocess but not until Blender has finished
+    p.wait()
```

Comment:
Glad this is now working! Have you tested this step when the package was installed normally and not in editable mode (`-e`)? Wondering if it works there as well.

Reply:
Never tried - doing that now

Reply:
Ran successfully!